tests/utils/node_launcher.go

```diff
@@ -0,0 +1,75 @@
+package utils
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/kurtosis-tech/kurtosis/api/golang/engine/lib/kurtosis_context"
+)
+
+const (
+	isPartitioningEnabled    = false
+	enclaveIdPrefix          = "test"
+	avalancheStarlarkPackage = "github.com/kurtosis-tech/avalanche-package"
+	// forces the node to launch on 9650 instead of ephemeral ports
+	forceExposeOn9650  = `{"test_mode": true}`
+	defaultParallelism = 4
+	firstNodeId        = "node-0"
+)
+
+func SpinupAvalancheNode() (error, string, func()) {
```

Comment:
The error should be the last return value instead of the first

Reply:
Yes, my bad. Fixed the ordering.